tests/testcases/core/libraries/rest_api/ModelDataTranslatorTest.php

```diff
@@ -467,8 +467,7 @@ public function testPrepareFieldValuesForJson($expected, $input, $field_obj)
     public function dataProviderForTestPrepareConditionsQueryParamsForModelsBad()
     {
         return array(
-            //case 0: array isn't numerically indexed
```

Comment:
I think it'd be handy to also put in the phpdocs for this unit test a link to the documentation that this unit-test covers.  I'll leave that up to you whether you do or not.

Reply:
Thanks for the suggestion. The unit tests in this file don't test an entire request and don't correspond to documentation examples.
I think the idea of making sure we test each documentation example is good though, but I think it should be a completely separate ticket. Also, it would be easiest to verify a code example has a corresponding unit test if those were link-to from the documentation (although that might make the documentation too cluttery). Anyways, I think that should be a separate ticket.